src/content/en/2021/caching.md

```diff
@@ -17,4 +19,328 @@ featured_stat_3: TODO
+
+Worth to mention if no caching headers are present in a response, then the browser is allowed to *heuristically* cache the response. Some may cache the response indefinitely, and some may not cache it at all. Because of this variation between browsers, it is essential to explicitly set specific caching rules to ensure that you control the cacheability of your content.
+
+### `Cache-Control` directives
+When using the `Cache-Control` header, you specify one or more *directives*—predefined values that indicate specific caching functionality. Multiple directives are separated by commas and can be set in any order, although some clash with one another (e.g., `public` and `private`). In addition, some directives take a value, such as `max-age`.
+
+Below is a table showing the most common `Cache-Control` directives:
+
+<figure>
+  <table>
+    <tr>
+     <th>Directive</th>
+     <th>Description</th>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">max-age</code></td>
+     <td>Indicates the number of seconds that a resource can be cached for relative to the current time. For example <code>max-age=86400</code>.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">public</code></td>
+     <td>Indicates that any cache can store the response, including the Browser and the CDN. This is assumed by default.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">no-cache</code></td>
+     <td>A cached resource must be revalidated before its use, via a conditional request, even if it is not marked as stale.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">must-revalidate</code></td>
+     <td>A stale cached entry must be revalidated before its use, via a conditional request.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">no-store</code></td>
+     <td>Indicates that the response must not be cached.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">private</code></td>
+     <td>The response is intended for a specific user and should not be stored by shared caches such as CDNs.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">immutable</code></td>
+     <td>Indicates that the cached entry will never change during its TTL and that revalidation is not necessary.</td>
+    </tr>
+  </table>
+  <figcaption>{{ figure_link(caption="<code>Cache-Control</code> directives.", sheets_gid="1944529311", sql_file="TODO") }}</figcaption>
+</figure>
```

Comment:
@rviscomi would you mind help create a figure similar to this one https://20211128t100318-dot-webalmanac.uk.r.appspot.com/static/images/2020/caching/cache-control-directives.png which helps show the Usage of Cache Control directives please, probably anything above 1% threshold.

I believe the data should be here https://docs.google.com/spreadsheets/d/1-v3yR0LZIC3t4zWtqTgR3jJsKjjRMP-HATU2caP8e2c/edit#gid=1944529311

Reply:
Added. Note that the `figcaption` above should be used to caption the table of definitions, separate from the other figure with the chart.

Reply:
Thank you